_data/banking.yml

```diff
@@ -119,6 +119,12 @@ websites:
       - hardware
     doc: https://www.bochk.com/en/security/2factorauthentication.html
 
+  - name: Bank of Scotland
+    url: https://www.bankofscotland.co.uk
+    img: bankofscotland.png
+    twitter: askbankofscot
+    facebook: bankofscotland
```

Comment:
```suggestion
    tfa:
      - totp
      - phone
      - sms
```

Reply:
Are you sure about "totp"? The confirmation via their app should rather be "proprietary". I see no sign of TOTP.

Reply:
@phallobst you're right. My bad

Reply:
Ok - Thanks for suggestions. I'll update the PR to include this plus also Lloyds & Halifax to the same if they are not already

Reply:
@rfadam try to keep one site listing per PR.